configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -1168,47 +1180,48 @@
         </div>
     </section>
 
-    <section id="transfer-out-section" sectionTag="fieldset"
-             headerTag="legend" headerStyle="title" headerCode="pihcore.transfer.title">
+    <section id="transferIn" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+             headerCode="pihcore.transferReferralIn">
         <div class="section-container-color">
-            <obsgroup groupingConceptId="PIH:13170">
+            <obsgroup groupingConceptId="PIH:13169">
                 <label>
-                    <uimessage code="pihcore.externalTransferQuestion" />
+                    <uimessage code="pihcore.externalTransferQuestion"/>
                 </label>
-                <p class="radio cb-align">
-                    <obs conceptId="PIH:HUM Disposition categories"
-                         answerConceptId="CIEL:160036" answerCode="emr.yes"
-                         style="checkbox" toggle="transfer-out-details"/>
+                <p class="side-by-side">
+                    <obs id="transfer-referral-spots"
+                         conceptId="PIH:REFERRAL OR TRANSFER" style="radio"
+                         answerConceptIds="PIH:11521,PIH:6965"
+                         answerCodes="pihcore.referral,pihcore.transfer"
+                         answerSeparator="" toggle="transfer-referral-out-details"/>
                 </p>
-
-                <div id="transfer-out-details">
-                    <p class="small">
+                <div id="transfer-referral-out-details">
+                    <p class="side-by-side">
                         <obs id="transfer-out-spots"
-                             conceptId="PIH:Transfer out location" style="radio"
```

Comment:
I don't think we want to change the concept here?  this is for the transfer out use caes, correct?